app/src/index.js

```diff
@@ -28,6 +30,38 @@ const handleSocketClose = () => {
   document.getElementById('stopRecordButton').disabled = true;
 };
 
+const getVideoCodecs = () => {
+  let params = new URLSearchParams(location.search.slice(1));
+  let videoCodec = params.get('videocodec')
+  if (!videoCodec) {
+    return {
+            kind: 'video',
+            mimeType: 'video/H264',
+            clockRate: 90000,
+            parameters: {
+              'packetization-mode': 1,
+              'profile-level-id': '4d0032',
+              'level-asymmetry-allowed': 1,
+              'x-google-start-bitrate': 1000
+            }
+          };
+  }
+  const codec = mediasoupConfig.router.mediaCodecs.find(c=>{
+     return ~c.mimeType.toLowerCase().indexOf(videoCodec.toLowerCase())
+  });
+  return codec !== undefined ? codec : {
```

Comment:
If codec is not defined, your returning the same as the above (if !videoCodec), would be better to return the default codec only once

Reply:
Yeah, it's two condition:

1. no query to set the `videocodec`
2. set the videocodec but invalid

Yes these lines look not good